src/revocation.rs

```diff
@@ -20,7 +20,7 @@ pub trait RevocableToken {
     /// Indicates the type of the token being revoked, as defined by [RFC 7009, Section 2.1](https://tools.ietf.org/html/rfc7009#section-2.1).
     ///
     /// Implementations should return `Some(...)` values for token types that the target authorization servers are
-    /// expected to know (e.g. because they are registered in the [OAuth Token Type Hints Registry](https://tools.ietf.org/html/rfc7009#section-4.1.2))
+    /// expected to know (e.g. because they are registered in the [OAuth Token Type Hints Registry](https://tools.ietf.org/html/rfc7009#section-4.1.2)
```

Comment:
I'm going to revert this one change since the trailing parenthesis corresponds to the one in the text preceding `e.g.`. that one is unterminated in the latest docs: https://docs.rs/oauth2/4.0.0-alpha.4/oauth2/revocation/trait.RevocableToken.html

Reply:
looks like you had already done that in #125 and then reverted your fix 😆